src/Microsoft.SqlTools.ServiceLayer/QueryExecution/Contracts/ExecuteRequests/QueryCompleteEvent.cs

```diff
@@ -23,6 +23,11 @@ public class QueryCompleteParams
         /// Summaries of the result sets that were returned with the query
         /// </summary>
         public BatchSummary[] BatchSummaries { get; set; }
+
+        /// <summary>
+        /// Process ID assigned to the query editor.
+        /// </summary>
+        public string PID { get; set; }
```

Comment:
This should be nullable too - we don't always set it (and other providers definitely won't necessarily be setting it so we need to be able to handle that)